src/StackAdmin/Azs.Storage.Admin/Module/Azs.Storage.Admin/Azs.Storage.Admin.psd1

```diff
@@ -6,142 +6,130 @@ Licensed under the MIT License. See License.txt in the project root for license
-    # DotNetFrameworkVersion = ''
+# Minimum version of Microsoft .NET Framework required by this module. This prerequisite is valid for the PowerShell Desktop edition only.
+# DotNetFrameworkVersion = ''
 
-    # Minimum version of the common language runtime (CLR) required by this module. This prerequisite is valid for the PowerShell Desktop edition only.
-    # CLRVersion = ''
+# Minimum version of the common language runtime (CLR) required by this module. This prerequisite is valid for the PowerShell Desktop edition only.
+# CLRVersion = ''
 
-    # Processor architecture (None, X86, Amd64) required by this module
-    # ProcessorArchitecture = ''
+# Processor architecture (None, X86, Amd64) required by this module
+# ProcessorArchitecture = ''
 
-    # Modules that must be imported into the global environment prior to importing this module
-    RequiredModules   = @(@{ModuleName = 'AzureRM.Profile'; ModuleVersion = '5.8.3'; },
-        @{ModuleName = 'AzureRM.Resources'; RequiredVersion = '6.4.3'; })
+# Modules that must be imported into the global environment prior to importing this module
+RequiredModules   = @(@{ModuleName = 'AzureRM.Profile'; ModuleVersion = '5.8.3'; },
+	@{ModuleName = 'AzureRM.Resources'; RequiredVersion = '6.4.3'; })
 
-    # Assemblies that must be loaded prior to importing this module
-    RequiredAssemblies = @('.\ref\fullclr\Microsoft.AzureStack.Management.Storage.Admin.dll')
+# Assemblies that must be loaded prior to importing this module
+RequiredAssemblies = @('.\ref\fullclr\Microsoft.AzureStack.Management.Storage.Admin.dll')
 
-    # Script files (.ps1) that are run in the caller's environment prior to importing this module.
-    # ScriptsToProcess = @()
+# Script files (.ps1) that are run in the caller's environment prior to importing this module.
+# ScriptsToProcess = @()
 
-    # Type files (.ps1xml) to be loaded when importing this module
-    TypesToProcess    = @('Generated.PowerShell.Commands\FormatFiles\Azs.Storage.Admin.Type.ps1xml')
+# Type files (.ps1xml) to be loaded when importing this module
+TypesToProcess = @()
 
-    # Format files (.ps1xml) to be loaded when importing this module
-    FormatsToProcess  = @('Generated.PowerShell.Commands\FormatFiles\Azs.Storage.Admin.Format.ps1xml')
+# Format files (.ps1xml) to be loaded when importing this module
+FormatsToProcess = @()
 
-    # Modules to import as nested modules of the module specified in RootModule/ModuleToProcess
-    NestedModules     = @()
+# Modules to import as nested modules of the module specified in RootModule/ModuleToProcess
+NestedModules = @()
 
-    # Functions to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no functions to export.
-    FunctionsToExport = 'Restore-AzsStorageAccount', 'New-AzsStorageQuota',
```

Comment:
Any reason why we have removed most of the cmdlets? Are there any replacement for these functionalities?

Reply:
Azs srp is refactored. Some cmdlets are moved to other components, some cmdlets are deprecated because nobody use them.